code/modules/clothing/suits/ego_gear/aleph.dm

```diff
@@ -81,7 +81,7 @@
 	desc = "It is not as unpleasant to wear as it is to look at. \
 	In fact, it seems to give you an illusion of comfort and bravery."
 	icon_state = "adoration"
-	armor = list(RED_DAMAGE = 70, WHITE_DAMAGE = 50, BLACK_DAMAGE = 70, PALE_DAMAGE = 50) // 240
+	armor = list(RED_DAMAGE = 80, WHITE_DAMAGE = 60, BLACK_DAMAGE = 40, PALE_DAMAGE = 60) // 240
```

Comment:
What..?

Reply:
It's just worse PL

Reply:
Kirie, *everything* is worse PL. That's how it is.
There is no reason to change it so hard.